docs/tutorial/ATutIntroduction.md

```diff
@@ -68,27 +68,5 @@ its capabilities.
 If Trick is not already installed on your machine, then you will need to do that
 first, by following the directions at: [Install Guide](/trick/documentation/install_guide/Install-Guide).
 
-Once Trick is installed on your machine, you will need add the Trick **bin**
```

Comment:
Why did you remove this section?

Reply:
This page already links to [the installation guide](https://github.com/Codym48/trick/blob/cleanup-docs/docs/documentation/install_guide/Install-Guide.md#4-optionally-update-your-environment), which seems to contain the latest and greatest information regarding environment variables and specifically advises against setting `TRICK_HOME` and friends,

I could restore a sentence about optionally modifying `PATH`?

Reply:
Ok, I put the content back but made it obvious that PATH modification is optional rather than required and left TRICK_HOME out.

Reply:
Thanks! We'll get this merged now